lib/pharos/exec_command.rb

```diff
@@ -11,13 +11,13 @@ class ExecCommand < Pharos::Command
     banner "Opens SSH sessions to hosts in the Kontena Pharos cluster."
 
     def execute
-      if command_list.empty?
-        signal_usage_error 'interactive mode can not be used with a non-interactive terminal' unless $stdin.tty? && $stdout.tty?
-        run_interactive
-        exit 0
-      end
-
       Dir.chdir(config_yaml.dirname) do
+        if command_list.empty?
```

Comment:
Config parsing in ssh/exec interactive mode was not done in `config_yaml.dirname` context.